nix/shell.nix

```
@@ -0,0 +1,26 @@
+{ pkgs, esy ? pkgs.nodePackages.esy, deku, npmPackages }:
+
+pkgs.mkShell {
+  shellHook = deku.configurePhase;
+
+  nativeBuildInputs = (with pkgs; [
+    # Make developer life easier
+    ## General tooling
+    docker
+    docker-compose
+    nodejs-12_x
+
+    # Nix files formatter
+    nixfmt
```

```suggestion
    nixfmt
    nodePackages.prettier
```
We also use prettier in the `dune build @fmt` target.

---

Hmm, I added it this way and that wasn't sufficient to have `prettier` on the path with direnv. `nix develop` works, but direnv doesn't. :thinking: